src/ert/_c_wrappers/enkf/model_config.py

```diff
@@ -24,46 +23,41 @@ def __init__(
         history_source: Optional[HistorySourceEnum] = None,
         runpath_format_string: Optional[str] = None,
         jobname_format_string: Optional[str] = None,
+        runpath_file: Optional[str] = None,
         gen_kw_export_name: Optional[str] = None,
         obs_config_file: Optional[str] = None,
         time_map_file: Optional[str] = None,
     ):
         self.num_realizations = num_realizations
         self.refcase = refcase
         self.data_root = data_root
-
         self.history_source = (
             history_source
             if self.refcase is not None and history_source is not None
             else self.DEFAULT_HISTORY_SOURCE
         )
+        self.jobname_format_string = (
+            replace_runpath_format(jobname_format_string) or "JOB<IENS>"
+        )
+        self.runpath_format_string = (
+            replace_runpath_format(runpath_format_string) or self.DEFAULT_RUNPATH
+        )
+        self.runpath_file = runpath_file
 
-        if runpath_format_string is None:
-            self.runpath_format_string = self.DEFAULT_RUNPATH
-        else:
-            self.runpath_format_string = replace_runpath_format(runpath_format_string)
-
-            if "%d" in runpath_format_string:
-                logger.warning(
-                    "RUNPATH keyword contains deprecated value placeholders, "
-                    f"`instead use: {self.runpath_format_string}`"
-                )
-            elif not any(x in runpath_format_string for x in ["<ITER>", "<IENS>"]):
+        if self.runpath_format_string is not None:
```

Comment:
Some simplification asked for by pylint here.

Reply:
I will do the simplification, but I think using two ifs makes reading the code much easier than figuring out the complex condition in one.